projects/unit/configs/all/shared_dec.yaml

```diff
@@ -0,0 +1,193 @@
+model_config:
+  unit:
+    base_args:
+      num_queries:
+        detection:
+          detection_coco: 100
+          detection_visual_genome: 100
+        vl:
+          vqa2: 25
+          # hateful_memes: 25
+          visual_entailment: 25
+        glue:
+          glue_qnli: 25
+          glue_qqp: 25
+          glue_sst2: 25
+          glue_mnli_mismatched: 25
+      share_decoders: true
+      decoder_hidden_dim: 768
+      dilation: true
+      use_task_embedding_in_img_encoder: true
+      use_task_embedding_in_lang_encoder: true
+    bert_config:
+      bert_model_name: bert-base-uncased
+    losses:
+    - logit_bce
+    # initialize the ResNet convnet backbone from DETR
+    base_ckpt_path: https://dl.fbaipublicfiles.com/detr/detr-r50-dc5-f0fb7ef5.pth
+    base_ckpt_load_backbone_only: true
+
+dataset_config:
+  vqa2:
```

Comment:
These are repeated in multiple configs. You should try to see if these can be moved to a default config which is included in other variants.

Reply:
(moved to a default config)